<html xmlns:string="xalan://java.lang.String" xmlns:lxslt="http://xml.apache.org/xslt">
<head>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Standard Error from JUnitTclTest</title>
</head>
<body>
<pre>WARNING: WebServer: Model stopped before executing actions (e.g. producing outputs) scheduled for execution at times [11.267000000000001, 12.011000000000001, 13.008000000000001]
WARNING: TriggeredRamp: Model stopped before executing actions (e.g. producing outputs) scheduled for execution at times [4.0]
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Failure executing the react function: Error: commonHost.js, react(), invoking a specific handler for "response": Exception occurred in input handler for accessor WebServer, which has now has been removed.  Exception was: java.lang.IllegalStateException: Response is closed Stacktrace was: undefined in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt; at line number 1695 at column number 24
  in .KeyValueStoreClient.WebServer
Because:
Error: commonHost.js, react(), invoking a specific handler for "response": Exception occurred in input handler for accessor WebServer, which has now has been removed.  Exception was: java.lang.IllegalStateException: Response is closed Stacktrace was: undefined in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt; at line number 1695 at column number 24
	at ptolemy.actor.lib.jjs.JavaScript._invokeMethodInContext(JavaScript.java:2223)
	at ptolemy.actor.lib.jjs.JavaScript.fire(JavaScript.java:1102)
	at ptolemy.domains.de.kernel.DEDirector._fire(DEDirector.java:1948)
	at ptolemy.domains.de.kernel.DEDirector.fire(DEDirector.java:502)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: javax.script.ScriptException: Error: commonHost.js, react(), invoking a specific handler for "response": Exception occurred in input handler for accessor WebServer, which has now has been removed.  Exception was: java.lang.IllegalStateException: Response is closed Stacktrace was: undefined in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt; at line number 1695 at column number 24
	at jdk.nashorn.api.scripting.NashornScriptEngine.throwAsScriptException(NashornScriptEngine.java:470)
	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeImpl(NashornScriptEngine.java:392)
	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeMethod(NashornScriptEngine.java:199)
	at ptolemy.actor.lib.jjs.JavaScript._invokeMethodInContext(JavaScript.java:2200)
	... 8 more
Caused by: /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt;:1695:24 Error: commonHost.js, react(), invoking a specific handler for "response": Exception occurred in input handler for accessor WebServer, which has now has been removed.  Exception was: java.lang.IllegalStateException: Response is closed Stacktrace was: undefined
	at jdk.nashorn.internal.objects.NativeError.initException(NativeError.java:137)
	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:102)
	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:106)
	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:110)
	at jdk.nashorn.internal.objects.NativeError.constructor(NativeError.java:129)
	at jdk.nashorn.internal.scripts.Script$Recompilation$4851$74558A$require.L:1#react#invokeSpecificHandler(/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt;:1695)
	at jdk.nashorn.internal.scripts.Script$Recompilation$4771$73967A$require.L:1#react(/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt;:1728)
	at jdk.nashorn.internal.runtime.ScriptFunctionData.invoke(ScriptFunctionData.java:639)
	at jdk.nashorn.internal.runtime.ScriptFunction.invoke(ScriptFunction.java:494)
	at jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:393)
	at jdk.nashorn.api.scripting.ScriptObjectMirror.callMember(ScriptObjectMirror.java:199)
	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeImpl(NashornScriptEngine.java:386)
	... 10 more
WARNING: WebServer: Model stopped before executing actions (e.g. producing outputs) scheduled for execution at times [11.067]
WARNING: JavaScriptBarrier: Model stopped before executing actions (e.g. producing outputs) scheduled for execution at times [18.0]
WARNING: WebServer: Model stopped before executing actions (e.g. producing outputs) scheduled for execution at times [11.085]
WARNING: JavaScriptBarrier: Model stopped before executing actions (e.g. producing outputs) scheduled for execution at times [19.0]
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Failure executing the react function: Error: commonHost.js, react(), invoking a specific handler for "response": Exception occurred in input handler for accessor WebServer, which has now has been removed.  Exception was: java.lang.IllegalStateException: Response is closed Stacktrace was: undefined in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt; at line number 1695 at column number 24
  in .WebServerBasic.WebServer
Because:
Error: commonHost.js, react(), invoking a specific handler for "response": Exception occurred in input handler for accessor WebServer, which has now has been removed.  Exception was: java.lang.IllegalStateException: Response is closed Stacktrace was: undefined in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt; at line number 1695 at column number 24
	at ptolemy.actor.lib.jjs.JavaScript._invokeMethodInContext(JavaScript.java:2223)
	at ptolemy.actor.lib.jjs.JavaScript.fire(JavaScript.java:1102)
	at ptolemy.domains.de.kernel.DEDirector._fire(DEDirector.java:1948)
	at ptolemy.domains.de.kernel.DEDirector.fire(DEDirector.java:502)
	at ptolemy.actor.CompositeActor.fire(CompositeActor.java:455)
	at ptolemy.actor.Manager.iterate(Manager.java:851)
	at ptolemy.actor.Manager.execute(Manager.java:369)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: javax.script.ScriptException: Error: commonHost.js, react(), invoking a specific handler for "response": Exception occurred in input handler for accessor WebServer, which has now has been removed.  Exception was: java.lang.IllegalStateException: Response is closed Stacktrace was: undefined in /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt; at line number 1695 at column number 24
	at jdk.nashorn.api.scripting.NashornScriptEngine.throwAsScriptException(NashornScriptEngine.java:470)
	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeImpl(NashornScriptEngine.java:392)
	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeMethod(NashornScriptEngine.java:199)
	at ptolemy.actor.lib.jjs.JavaScript._invokeMethodInContext(JavaScript.java:2200)
	... 8 more
Caused by: /home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt;:1695:24 Error: commonHost.js, react(), invoking a specific handler for "response": Exception occurred in input handler for accessor WebServer, which has now has been removed.  Exception was: java.lang.IllegalStateException: Response is closed Stacktrace was: undefined
	at jdk.nashorn.internal.objects.NativeError.initException(NativeError.java:137)
	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:102)
	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:106)
	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:110)
	at jdk.nashorn.internal.objects.NativeError.constructor(NativeError.java:129)
	at jdk.nashorn.internal.scripts.Script$Recompilation$11697$74558A$require.L:1#react#invokeSpecificHandler(/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt;:1695)
	at jdk.nashorn.internal.scripts.Script$Recompilation$11647$73967A$require.L:1#react(/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt;:1728)
	at jdk.nashorn.internal.runtime.ScriptFunctionData.invoke(ScriptFunctionData.java:639)
	at jdk.nashorn.internal.runtime.ScriptFunction.invoke(ScriptFunction.java:494)
	at jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:393)
	at jdk.nashorn.api.scripting.ScriptObjectMirror.callMember(ScriptObjectMirror.java:199)
	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeImpl(NashornScriptEngine.java:386)
	... 10 more
HttpClientHelper.java: Received an error for request 0: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: localhost/127.0.0.1:8597
io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: localhost/127.0.0.1:8597
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:717)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:325)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:340)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:633)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:580)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:497)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:459)
	at io.netty.util.concurrent.SingleThreadEventExecutor$5.run(SingleThreadEventExecutor.java:886)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.net.ConnectException: Connection refused
	... 11 more
java.lang.RuntimeException: Execution failed
	at ptolemy.moml.MoMLSimpleApplication$UnloadThread.run(MoMLSimpleApplication.java:402)
Caused by: ptolemy.kernel.util.IllegalActionException: Exceptions occurred during wrapup:
ptolemy.kernel.util.IllegalActionException: Failure executing the wrapup function: Error: TrainableTest: The input handler of this accessor was never invoked. Usually, this is an error indicating that starvation is occurring. in &lt;function&gt; at line number 393 at column number 16
  in .WebServerTimeout.TrainableTest
Because:
Error: TrainableTest: The input handler of this accessor was never invoked. Usually, this is an error indicating that starvation is occurring. in &lt;function&gt; at line number 393 at column number 16
ptolemy.kernel.util.IllegalActionException: Failure executing the wrapup function: Error: TrainableTest2: The input handler of this accessor was never invoked. Usually, this is an error indicating that starvation is occurring. in &lt;function&gt; at line number 393 at column number 16
  in .WebServerTimeout.TrainableTest2
Because:
Error: TrainableTest2: The input handler of this accessor was never invoked. Usually, this is an error indicating that starvation is occurring. in &lt;function&gt; at line number 393 at column number 16

  in .WebServerTimeout.DE Director
Because:
Failure executing the wrapup function: Error: TrainableTest: The input handler of this accessor was never invoked. Usually, this is an error indicating that starvation is occurring. in &lt;function&gt; at line number 393 at column number 16
  in .WebServerTimeout.TrainableTest
Because:
Error: TrainableTest: The input handler of this accessor was never invoked. Usually, this is an error indicating that starvation is occurring. in &lt;function&gt; at line number 393 at column number 16
	at ptolemy.actor.Director.wrapup(Director.java:1736)
	at ptolemy.domains.de.kernel.DEDirector.wrapup(DEDirector.java:1526)
	at ptolemy.actor.CompositeActor.wrapup(CompositeActor.java:2735)
	at ptolemy.actor.Manager.wrapup(Manager.java:1512)
	at ptolemy.actor.Manager.execute(Manager.java:407)
	at ptolemy.actor.Manager.run(Manager.java:1263)
	at ptolemy.actor.Manager$PtolemyRunThread.run(Manager.java:1915)
Caused by: ptolemy.kernel.util.IllegalActionException: Failure executing the wrapup function: Error: TrainableTest: The input handler of this accessor was never invoked. Usually, this is an error indicating that starvation is occurring. in &lt;function&gt; at line number 393 at column number 16
  in .WebServerTimeout.TrainableTest
Because:
Error: TrainableTest: The input handler of this accessor was never invoked. Usually, this is an error indicating that starvation is occurring. in &lt;function&gt; at line number 393 at column number 16
	at ptolemy.actor.lib.jjs.JavaScript._invokeMethodInContext(JavaScript.java:2223)
	at ptolemy.actor.lib.jjs.JavaScript.wrapup(JavaScript.java:2118)
	at ptolemy.actor.Director.wrapup(Director.java:1722)
	... 6 more
Caused by: javax.script.ScriptException: Error: TrainableTest: The input handler of this accessor was never invoked. Usually, this is an error indicating that starvation is occurring. in &lt;function&gt; at line number 393 at column number 16
	at jdk.nashorn.api.scripting.NashornScriptEngine.throwAsScriptException(NashornScriptEngine.java:470)
	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeImpl(NashornScriptEngine.java:392)
	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeMethod(NashornScriptEngine.java:199)
	at ptolemy.actor.lib.jjs.JavaScript._invokeMethodInContext(JavaScript.java:2200)
	... 8 more
Caused by: &lt;function&gt;:393:16 Error: TrainableTest: The input handler of this accessor was never invoked. Usually, this is an error indicating that starvation is occurring.
	at jdk.nashorn.internal.objects.NativeError.initException(NativeError.java:137)
	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:102)
	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:106)
	at jdk.nashorn.internal.objects.NativeError.&lt;init&gt;(NativeError.java:110)
	at jdk.nashorn.internal.objects.NativeError.constructor(NativeError.java:129)
	at jdk.nashorn.internal.scripts.Script$Recompilation$14469$18508$\^function\_.L:1#wrapup(&lt;function&gt;:393)
	at jdk.nashorn.internal.scripts.Script$Recompilation$14467$30328$require.L:1#Accessor#wrapup(/home/travis/build/icyphy/ptII/ptolemy/actor/lib/jjs/external/require.js#401:30&lt;eval&gt;:673)
	at jdk.nashorn.internal.runtime.ScriptFunctionData.invoke(ScriptFunctionData.java:637)
	at jdk.nashorn.internal.runtime.ScriptFunction.invoke(ScriptFunction.java:494)
	at jdk.nashorn.internal.runtime.ScriptRuntime.apply(ScriptRuntime.java:393)
	at jdk.nashorn.api.scripting.ScriptObjectMirror.callMember(ScriptObjectMirror.java:199)
	at jdk.nashorn.api.scripting.NashornScriptEngine.invokeImpl(NashornScriptEngine.java:386)
	... 10 more
</pre>
</body>
</html>
